spec/system/admin/instrument_price_policies_controller_spec.rb

```diff
@@ -400,4 +400,115 @@
       expect(page).not_to have_content("$90.00") # Step 3 rate (removed)
     end
   end
+
+  context "Schedule Rule Daily pricing mode" do
+    let(:pricing_mode) { Instrument::Pricing::SCHEDULE_DAILY }
+
+    it "editting rates automatically fills adjustments", :js do
+      visit new_facility_instrument_price_policy_path(facility, instrument)
+
+      check("price_policy_#{base_price_group.id}[can_purchase]")
+      check("price_policy_#{cancer_center.id}[can_purchase]")
+
+      base_rate = "100"
+      fill_in("price_policy_#{base_price_group.id}[usage_rate_daily]", with: base_rate)
+
+      find_field(
+        "price_policy_#{cancer_center.id}[usage_rate_daily]",
+        type: "hidden"
+      ).tap do |hidden_input|
+        expect(hidden_input.value).to eq(base_rate)
+      end
+    end
+
+    it "can add price policies" do
+      visit new_facility_instrument_price_policy_path(facility, instrument)
+
+      expect(page).to have_text("Rate Per Day")
+      expect(page).to_not have_text("Rate Per Hour")
+
+      expect(page).to_not have_text("Reservation Cost")
+      expect(page).to_not have_text("Minimum Cost")
+
+      PriceGroup.find_each do |price_group|
+        expect(page).to_not have_field("price_policy_#{price_group.id}[minimum_cost]")
+        expect(page).to_not have_field("price_policy_#{price_group.id}[cancellation]")
+        expect(page).to_not have_field("price_policy_#{price_group.id}[usage_rate]")
+      end
+
+      fill_in("note", with: "Some note about pricing")
+
+      check("price_policy_#{base_price_group.id}[can_purchase]")
+      check("price_policy_#{cancer_center.id}[can_purchase]")
+      uncheck("price_policy_#{external_price_group.id}[can_purchase]")
+
+      fill_in("price_policy_#{base_price_group.id}[usage_rate_daily]", with: "100")
+      fill_in("price_policy_#{cancer_center.id}[usage_subsidy_daily]", with: "30")
+
+      # this is autofilled by js when enabled
```

Comment:
If you enable JS for this spec, and add an expectation for the daily rate displayed to the user to be 100, I think you could delete the spec above which starts on line 407.  what do you think?